Extras/Scripts/sign-bundled.sh

```diff
@@ -90,7 +90,7 @@ sign_folder()
   local FOLDER="$1"
   local NAME=`basename "$FOLDER/"`
   if [ -e "$FOLDER/" ]; then
-    #echo "Signing $NAME as: ${CODE_SIGN_IDENTITY}"
+    #echo "Signing $NAME as: ${EXPANDED_CODE_SIGN_IDENTITY}"
```

Comment:
Then for these usages, we don't want the `CODE_SIGN_IDENTITY` since it's a bit vague. It'll be `Mac Developer` which if you have more than one matching certificate (e.g. you're a member of multiple teams), Xcode can't resolve.

Instead, by using `EXPANDED_CODE_SIGN_IDENTITY` this is the full hash of the certificate that Xcode has already decided best matches the identity name, _and_ the team.